src/gui/optionsdialog.ui

```diff
@@ -90,7 +90,7 @@
      </widget>
      <widget class="QStackedWidget" name="tabOption">
       <property name="currentIndex">
-       <number>0</number>
+       <number>3</number>
```

Comment:
All these changes down to line 1803 are made by Designer and i was often told to undo them, just a heads up. If you manage to stop it from doing them i'm very interested to know how.

Reply:
Oh crap... I guess I'll just have to diff it with older revision.